circle.yml

```diff
@@ -38,6 +40,14 @@ dependencies:
     # create a version.json
     - printf '{"commit":"%s","version":"%s","source":"https://github.com/%s/%s"}\n' "$CIRCLE_SHA1" "$CIRCLE_TAG" "$CIRCLE_PROJECT_USERNAME" "$CIRCLE_PROJECT_REPONAME" > version.json
 
+    # build addon
+    - cd addon/
```

Comment:
Something else I think I just learned after some googling of other folks' CircleCI usage: Each of these entries runs in the root directory of the project. It's not like a continuous shell script - each line is its own separate script. So, `cd` is undone right after you issue it.

Probably needs an ugly one-liner something more like `cd addon && npm config set spin false && npm install && npm run lint && npm run sign`